docs/locales/zh_CN/LC_MESSAGES/tutorial/Federate_Learning_for_Image_Classification.po

```diff
@@ -109,109 +109,114 @@ msgid ""
 msgstr ""
 "隐语提供了多种聚合方案，SecureAggregator和PPUAggregator可用于安全聚合，更多安全聚合方案可以参考 `安全聚合 "
-"<../developer/algorithm/secure_aggregation.ipynb>`__."
+"<../developer/algorithm/secure_aggregation.ipynb>`__。"
 
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:452
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:453
 msgid "Define ``FLModel``"
 msgstr "定义联邦模型"
 
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:478
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:479
 msgid "Lets run model"
 msgstr "跑起来"
 
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:642
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:643
 msgid "Contrast experiment to local training"
 msgstr "对比单方模型"
 
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:654
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:655
 msgid "Model"
 msgstr "模型"
 
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:656
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:657
+msgid "The model structure is consistent with the fl model above."
+msgstr "模型结构和上面fl的模型保持一致。"
+
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:669
+msgid "Data"
+msgstr "数据"
+
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:671
 msgid ""
-"The model structure is consistent with the fl model above #### Data Here,"
-" we only used data after a horizontal segmentation, with a total of "
-"20,000 samples for ``Alice``"
-msgstr "模型结构和上面fl的模型保持一致#### 数据数据同样使用mnist数据集，单方模型这里我们只是用了切分后的alice方数据共20000个样本"
+"Here, we only used data after a horizontal segmentation, with a total of "
+"20,000 samples for ``Alice``."
+msgstr "数据同样使用mnist数据集，单方模型这里我们只是用了切分后的Alice方数据共20000个样本。"
 
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:837
+#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:852
 msgid ""
 "The two experiments above simulated a training problem in a typical "
-"horizontal federation scenario, \\* Alice and Bob have same type of data "
-"\\* Each side had only a portion of the sample, but the training "
-"objectives is the same If Alice only uses her own data to train the "
-"model, could only obtain a model with an accuracy of 0.945. However, if "
-"Bob’s data is combined, a model with an accuracy close to 0.995 can be "
-"obtained. In addition, the generalization performance of the model "
-"jointly trained with multi-party data will also be better"
-msgstr ""
-"上面两个实验模拟了一个典型的水平联邦场景的训练问题，\\* alice和bob拥有类型的图片\\* "
-"每一方只有样本的一部分数据，但是双方的训练目的是一致的如果alice只用自己的一方数据来训练模型，能够得到一个精确度0.945的模型，但是如果联合bob的数据之后，可以获得一个精确度接近0.995的模型，而且多方数据联合训练的模型的泛化性能也会更好"
-
-#: ../../tutorial/Federate_Learning_for_Image_Classification.ipynb:850
+"horizontal federation scenario, Alice and Bob have same type of data. "
+"Each side had only a portion of the sample, but the training objectives "
```

Comment:
Here is a display issue， the two lines with "*" are intended to be displayed as list. I will fix this problem later. thanks for your work